using-virtual-machines/virtual-machine-replica-set.adoc

```diff
@@ -219,3 +219,48 @@ Events:
 `Replicas` is `3` and `Ready Replicas` is `2`. This means that at the
 moment when showing the status, three Virtual Machines were already
 created, but only two are running and ready.
+
+Scaling via the Scale Subresource
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+______________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+*Note:* This requires the `CustomResourceSubresources` feature gate to be
+enables for clusters prior to 1.11.
+______________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+
+The `VirtualMachineInstanceReplicaSet` supports the `scale` subresource. As a
+consequence it is possible to scale it via `kubectl`:
+
+[source,bash]
+----
+$ kubectl scale vmirs myvmirs --replicas 5
+----
+
+Using the Horizontal Pod Autoscaler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+______________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+*Note:* This requires at cluster newer or equal to 1.11.
+______________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+
+The `HorizontalPodAutoscaler` (HPA) can be used with a
```

Comment:
Adding the link on the official documentation can be useful

Reply:
Done.